bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/interactive/ConvertTask.java

```diff
@@ -0,0 +1,208 @@
+     * 2. Paths to models required for the origin model
+     * 3. Path to the folder the result should be saved in
+     * 4. File name of the resulting model files
+     * @param args Command line parameters
+     */
+    public static void main(String[] args) {
+        if (args.length > 0) {
+            handleParameters(args);
+            System.exit(0);
+        }
+        handleInteractive();
+        System.exit(0);
+    }
+
+    private static void handleParameters(String[] parameters) {
+        String[] conversionSplit = parameters[0].split("2");
+        if (conversionSplit.length != 2) {
+            System.err.println("First parameter must describe the desired conversion. Example: pcm2dfd");
+            System.exit(-1);
+        }
+        Optional<ModelType> origin = ModelType.fromAbbreviation(conversionSplit[0]);
+        if (origin.isEmpty()) {
+            System.err.println("Invalid requested conversion origin: " + conversionSplit[0]);
+            System.exit(-1);
+        }
+        Optional<ModelType> destination = ModelType.fromAbbreviation(conversionSplit[1]);
+        if (destination.isEmpty()) {
+            System.err.println("Invalid requested conversion destination: " + conversionSplit[1]);
+            System.exit(-1);
+        }
+        List<String> parameterList = new ArrayList<>(Arrays.stream(parameters).toList());
+        parameterList.remove(0);
+
+        ConversionTable conversionTable = new ConversionTable();
+        Converter converter = conversionTable.getConverter(ConversionKey.of(origin.get(), destination.get()));
+        ConverterModel converterModel = getConverterModel(origin.get(), parameterList);
+
+        PersistableConverterModel persistableConverterModel = converter.convert(converterModel);
+
+        if (parameterList.size() != 2) {
+            System.err.println("Need two additional parameter for saving the converted model, but got: " + parameterList.size());
+            System.exit(-1);
+        }
+        String filePath = parameterList.remove(0);
+        String fileName = parameterList.remove(0);
+        persistableConverterModel.save(filePath, fileName);
+    }
+
+    private static void handleInteractive() {
+        Scanner scanner = new Scanner(System.in);
+        ModelType origin = getOrigin(scanner);
+        ModelType goal = getGoal(scanner, origin);
+        runConversion(origin, goal, scanner);
+    }
+
+    private static ModelType getOrigin(Scanner scanner) {
+        System.out.println("Please enter the desired model: ");
+        ConversionTable conversionTable = new ConversionTable();
+        List<ModelType> origins = conversionTable.getPossibleOrigins()
+                .stream()
+                .toList();
+        return getModelTypeFromSelection(scanner, origins);
+    }
+
+    private static ModelType getGoal(Scanner scanner, ModelType origin) {
+        System.out.println("Please enter the desired model: ");
```

Comment:
Two times desried model (see line 88) should be something like input / output model(s). When using it the first time, I already accidentally mixed the up :D